server/server.go

```diff
@@ -590,8 +590,10 @@ func (s *Server) Index(w http.ResponseWriter, _ *http.Request) {
 func (s *Server) Healthz(w http.ResponseWriter, _ *http.Request) {
 	data, err := json.MarshalIndent(&struct {
 		Status string `json:"status"`
+		Version string `json:"version"`
```

Comment:
Can you make this `AtlantisVersion`? I can see us also exposing Terraform version so it would be good to be able to disambiguate.

Reply:
Done